packages/auth-middleware/README.md

```diff
@@ -0,0 +1,39 @@
+# Auth middleware
+
+## Installation
+
+Using [yarn](https://yarnpkg.com/lang/en/):
+
+    $ yarn add @tsh/auth-middleware
+
+Using [npm](https://www.npmjs.com/):
+
+    $ npm install --save @tsh/auth-middleware
+
+Then, to enable authMiddleware, use [`applyMiddleware`](https://redux.js.org/api-reference/applymiddleware):
+
+```js
+import { applyMiddleware, createStore } from 'redux';
+
+import { apiMiddleware } from 'redux-api-middleware';
+import { authMiddleware } from '@tsh/auth-middleware';
+
+import { appReducer } from 'app/app.reducer';
+
+const middlewares = [authMiddleware, apiMiddleware];
+
+const store = createStore(appReducer, applyMiddleware(...middlewares));
+```
+
+and add authReducer to your app:
+
+```js
+import { combineReducers } from 'redux'
+import { authReducer } from '@tsh/auth-middleware';
+import ...
+
+export default combineReducers({
+  authReducer,
```

Comment:
Maybe it can be added under `auth` property?

Reply:
What do you mean?

Reply:
Łukasz meant to name it `{auth: authReducer}`